punchcut/src/source/glyf.rs

```diff
@@ -0,0 +1,109 @@
+/*!
+TrueType outlines loaded from the `glyf` table.
+
+*/
+
+mod math;
+mod outline;
+mod scaler;
+
+pub use outline::Outline;
+pub use scaler::Scaler;
+
+pub use read_fonts::tables::glyf::Point;
+
+/// Context for loading for TrueType glyphs.
```

Comment:
The term context throws me here, this is a fairly raw representation of an actual glyph isn't it?

Reply:
It represents the intermediate state that is necessary for loading a glyph. This is where the hinting cache will live so it appears sparse at the moment and may seem less "context-like" at a glance.